app/controllers/providers/means/cash_incomes_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Providers
   module Means
     class CashIncomesController < ProviderBaseController
-      before_action :aggregated_cash_income, only: %i[show update]
+      before_action :setup_variables, only: %i[show update]
```

Comment:
**nit** feel free to ignore. for grepability perhaps something less generic like `setup_cash_income[_var]s`